candi/cloud-providers/azure/docs/LAYOUTS.md

```diff
@@ -4,15 +4,17 @@ title: "Cloud provider — Azure: Layouts"
 
 > **Caution!** Only [regions](https://docs.microsoft.com/en-us/azure/availability-zones/az-region) where `Availability Zones` are available are supported.
 
-## Layouts
-### Standard
```

Comment:
@Zhbert We need this header to link to it from e.g. getting started. Let's leave it as it was?

Reply:
@z9r5 This is not entirely logical if there is only one scheme... But if you need a link, then let's leave it.

Reply:
I think this is not a guide, and there will be nothing wrong if we leave only one scheme with thr one header.